src/index.ts

```diff
@@ -0,0 +1,32 @@
+import { getSizeClass } from "./utils/sizeClasses.js"
+import "./utils/webFontLoader.js"
+
+import * as components from './components'
+import * as directives from './directives'
+
+const initializeCustomProps = (tokens = [], prefix) => {
+  for (const param in tokens) {
+    ;(document.querySelector(":root") as HTMLElement).style.setProperty("--oc-" + prefix + param, tokens[param])
+  }
+}
+
+export default {
+  install(Vue, options: any = {}) {
+    const themeOptions = options.tokens
+    initializeCustomProps(themeOptions?.breakpoints, "breakpoint-")
+    initializeCustomProps(themeOptions?.colorPalette, "color-")
+    initializeCustomProps(themeOptions?.fontSizes, "font-size-")
+    initializeCustomProps(themeOptions?.sizes, "size-")
+    initializeCustomProps(themeOptions?.spacing, "space-")
+
+    Object.values(components).forEach(c => Vue.component(c.name, c))
+    Object.values(directives).forEach(d => Vue.directive(d.name, d))
+  }
+}
+
+export const utils = { getSizeClass }
+export * as components from './components'
+export * as composables from './composables'
+export * as directives from './directives'
+export * as helpers from './helpers'
+export * as mixins from './mixins'
```

Comment:
I'm not 100% sure this is exactly the same export structure as before but it's standard and we should do it this way IMHO.

Rollup spits out a warning that we are exporting a default and named modules, it says we should add `output.exports: "named"` and that it would disable the warning. Yes, it does get rid of the warning ... and the default export 😒 
As I would like to switch to Vite in a follow up PR I don't care too much about the warning.